modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,121 @@
+* If you select a saved vCenter instance, the wizard connects to the vCenter instance using the saved credentials.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+. In the *Connect to Provider* screen, select *Red Hat Virtualization (RHV)* from the *Provider* list.
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>/ovirt-engine/api`
+** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate or paste the CA certificate into the field.
++
+You can obtain the RHV Manager CA certificate with the following command:
++
+----
+$ curl \
+--output ca.crt \
+'http://<RHV_Manager_FQDN>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA'
+----
+
+** *Username*: RHV Manager user name in the following format: `<username>@<domain>`
+** *Password*: RHV Manager password
+
+* If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
+endif::[]
+
+. Click *Check and Save* and wait for the connection to complete.
+
+ifdef::virt-importing-rhv-vm[]
+. Select a cluster and a virtual machine to import.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+. Select a virtual machine or a template to import.
+endif::[]
+
+. Click *Next*.
+. In the *Review* screen, review your settings.
+
+ifdef::virt-importing-rhv-vm[]
+. Optional: You can select *Start virtual machine on creation*.
+endif::[]
+
+. Optional: Click *Edit* to update the following settings:
+
+* *General*:
+
+ifdef::virt-importing-rhv-vm[]
+** *Name*: You must update the virtual machine name if the same name is being used by another virtual machine in the same namespace.
+** *Description*: Optional
+
+* *Storage* > *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class, Cinder, which is not supported for importing a RHV virtual machine. If a default storage class is not specified, the virtual disk's PVC is bound to a PV without a specified storage class.
+
+* *Networking* > *Network*: You can select a network from a list of available `NetworkAttachmentDefinition` objects.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+** *Name*: You must update the virtual machine name if the same name is being used by another virtual machine in the same namespace.
+** *Description*: Optional
+** *Operating System*
+** *Flavor*
+** *Memory*
+** *CPUs*
+** *Workload Profile*
+
+* *Networking*:
+
+** *Name*
+** *Model*
+** *Network*
+** *Type*
+** *MAC Address*
++
+Optional: Click *Add Network Interface* to add an interface.
+
+* *Storage*:
+
+** *Name*
+** *Source*: For example, *Import Disk*
+** *Size*
+** *Interface*
+** *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class to create the virtual machine.
```

Comment:
Need to indicate cases that if not selected, need to be aware of what is the default.
Because standard Cinder, and Ceph-rhb (block based) are not supported,
and it might be that the default was removed.